main.cpp

```diff
@@ -74,67 +174,127 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+float triangleArea(float base = 1.0f, float height = 5.0f)
+{ 
+    ignoreUnused(base, height);
+    return {}; 
+}  
 
 /*
  2)
  */
+float minutesToHours(int minute = 60)
+{ 
+    ignoreUnused(minute);
+    return {}; 
+}  
 
 /*
  3)
  */
+int countZeros(float number = 10000.0f)
+{ 
+    ignoreUnused(number);
+    return {}; 
+}   
 
 /*
  4)
  */
+bool isZero(long number = 20)
+{ 
+    ignoreUnused(number);
+    return {}; 
+}  
 
 /*
  5)
  */
+int findMax(int firstNum, int secondNum, int thirdNum)
+{ 
+    ignoreUnused(firstNum,secondNum,thirdNum);
+    return {}; 
+}   
 
 /*
  6)
  */
+int measureDistance(int steps, float stepSize)
+{ 
+    ignoreUnused(steps, stepSize);
+    return {}; 
+} 
+
 
 /*
  7)
  */
+bool isLargerThanZero(double myDoubleNum)
+{ 
+    ignoreUnused(myDoubleNum);
+    return {}; 
+} 
+
 
 /*
  8)
  */
+void startCounter(int initVal = 0)
+{ 
+    ignoreUnused(initVal);
+}  
 
 /*
  9)
  */
+void stopCounting(bool stopFlag = false)
+{ 
+    ignoreUnused(stopFlag);
+}  
 
 /*
  10)
  */
+ void playAudioForNSeconds(int seconds = 1)
+{ 
+    ignoreUnused(seconds);
+} 
 
 int main()
 {
     //example of calling that function
     rentACar(6, 2); 
     
     //1)
+    triangleArea(2.0f, 3.0f);
     
     //2)
+    minutesToHours(10);
     
     //3)
+    countZeros(100000.0f);
     
     //4)
+    isZero(1);
     
     //5)
+    findMax(2, 3, 4);
     
     //6)
+    measureDistance(10, 0.3f);
     
     //7)
+    isLargerThanZero(0.0001);
     
     //8)
+    startCounter(1);
     
     //9)
+    stopCounting(true);
     
     //10)
+    playAudioForNSeconds(10);
+
```

Comment:
I would like you to store the returned value from any function that returns a value into a local variable. 
At the end of main(), before the "good to go", pass all of these local variables to a single call of `ignoreUnused()`

- [ ] please revise according to instructions above

Reply:
I made the changes.